### DISCUSSION TITLE
How to submit front and back side of Student ID in one picture? I have my name & photo on front, and ID expiry date on back side.

### DISCUSSION THREAD

**Discussion by TestCode2080** [community/community#131238] [GitHub Education]
### Hi everyone,

I am having a very hard time uploading my ID. My photo, school name, registration number is on the front side, and ID issue date, expiry date, school phone number and address on the back side of the card. How am I suppose to submit both together? I just submitted back side, and my application got rejected instantly. Now I have submitted the front side, no reply so far. This student verification process is not very well thought out.

**Comment by davevad93** [comment]
Hi @TestCode2080  ,

First take a photo of both sides of the ID card, then use some software (there are several available online) to combine them into a single `png` or `jpg` file and finally upload it to the application form.

**Reply by TestCode2080** [reply]
There is no file upload option, otherwise I would have done it. Only option is to take a picture from a
webcam or phone.

**Reply by davevad93** [reply]
That's weird, because I was able to upload the required documentation when I applied last year, lately I've seen other users saying that there was no option to upload it in the application form, I don't know why. Try another browser and see if you have any luck.

**Reply by TestCode2080** [reply]
There was a file upload option available when I applied 2 years ago as well. Now, they have changed the process. I was re-verifying myself today. There is no "file upload" option available anymore.

**Reply by davevad93** [reply]
The thing is that for some people the upload option appears but it doesn't work, it just refreshes the page, while for most it doesn't even show up, That's something that GitHub has to fix as soon as possible, because it doesn't make any sense...

**Reply by henilmalaviya** [reply]
@TestCode2080 i had same issue. there is actually a "Cancel Application" (or something like that) Button at the bottom of upload page. I cancelled the application and re-applied again from desktop or laptop device and it was showing "Upload" option. This might help you.

**Reply by TestCode2080** [reply]
I was already using my laptop to re-verify. No upload option.

**Reply by jasonhall** [reply]
I did that and it was rejected because the image "appears to be edited. Manipulated images may not be used." I scanned both sides of my student card, then just copied one side to be on the same image as the other and sent it as a single file. It was rejected and my current status became "expired" before I have a chance to try another document.
